### DISCUSSION TITLE
Dev stuff???

### DISCUSSION THREAD

**Discussion by ThatLegend** [Prodigy-Hacking/ProdigyMathGameHacking#1175] [Questions] [answered]
Hello Developer or person who made Phex I would like to help you guys make the extension better. and here is proof that I'm a good developer. (my google developer profile):
![Screenshot 2021-11-09 202746](https://user-images.githubusercontent.com/94025261/141032904-c33a0dc1-0920-4323-93b5-b5315a9932bb.jpg)

Thanks! 
P.S Get back to me when you can

**Comment by PatheticMustan** [comment, marked as answer]
Hey! If you would like to talk more, we mainly discuss things on the [PMGH Discord](https://discord.gg/XQDfbfq).

**Reply by ThatLegend** [reply]
My dizzy is banned (rip)

**Reply by ThatLegend** [reply]
I have fought with them for 3 months now.
